Huge revamp of ru_ru localization + minor en_en fix

JustS-js opened the pull request:
Love doing pr's at 6 am

## What have changed:
### ru_ru
- Translated missing entries
- Translated lore
- Fixed lots and lots and LOTS of spelling and grammatical mistakes, such as missing declensions, cases and other russian-specific oddities
- Established parity with vanilla ru_ru localization (blocks and items have a specific naming pattern)
- Fixed context-based mistakes
- Probably introduced new spelling mistakes? dunno i'm not a god
### en_en
- Changed patchouli entry for division action, "dividend" and "divisor" were swapped. Simplified to avoid unnecessary complexity

object-Object commented:
Thanks for the huge contribution!

Would you be able to rebase this into less commits? At a minimum, one for en_us and one for ru_ru, but a few more would also be fine if there are logical places to split up the changes.

Also, I think all of the current check failures should be fixed by merging in the changes from `main`, but you'll probably want to wait until after rebasing to do that.

Sorry for the late review.

object-Object reviewed (changes requested):
(see other comment, it won't let me request changes without typing something here)

JustS-js commented:
Rebased and squashed all commits into [e96930d](https://github.com/FallingColors/HexMod/pull/832/commits/e96930df15077614e9e062de4e23e6662913eff4) and [2fa621d](https://github.com/FallingColors/HexMod/pull/832/commits/2fa621d1122f624797766cfeaca40bb052fdc3fe) and then merged new changes from main into this branch. Also translated itemgroup for scrolls

LedinecMing commented:
@JustS-js стоит обдумать лучшее слова для перевода, нежели Хекс, оно выглядит неуместно

LedinecMing commented:
Thanks for the huge amount of improvements.

JustS-js commented:
@LedinecMing The word "Hex" has two major meanings in english, something with 6 edges and something related to witchery and magic. I am not sure that there is something similar in russian. I am thinking that it would be too arrogant of me to drastically change the actual _**name of the mod itself**_!
But I understand the problem you mentioned. In the places where the meaning of word is certain (for example, where it is implied that "Hex" means the spell) it would be better to change it accordingly! I will rework that part of the translation 👍

LedinecMing commented:
Ancient cypher chapter is not translated

LedinecMing reviewed (approved):
Tried to fix everything mentioned with my pr

LedinecMing commented:
@JustS-js bump

LedinecMing commented:
@JustS-js bump

object-Object commented:
Seems like there are some merge conflicts to resolve now?

JustS-js commented:
@object-Object Hi, I resolved the conflict and synced translation with your latest changes to en_en

object-Object commented on Common/src/main/resources/assets/hexcasting/lang/ru_ru.flatten.json5:
```diff
@@ -1684,88 +1864,95 @@
+          title: "Дополнительные записи",
+          "1": "я могу избежать специального поведения $(l:patterns/patterns_as_iotas#hexcasting:open_paren)$(action)Интро-/$ и $(l:patterns/patterns_as_iotas#hexcasting:close_paren)$(action)Ретроспекций/$, начертив $(l:patterns/patterns_as_iotas#hexcasting:escape)$(action)Рассмотрение/$ перед ними, что просто добавит их в список, не влияя на количество $(l:patterns/patterns_as_iotas#hexcasting:c ...
+          "2": "Если йота, не являющаяся руной, присутствует в списке на исполнение руной $(l:patterns/meta#hexcasting:eval)$(action)гамбита Гермеса/$ (или любой другой мета-руной исполнения), то произойдёт оишбка. Но я могу избежать этого, используя руны, описаные ниже.$(br2)Прямо как с йотами рун, руны ниже позволяют \"экранировать\" $(o)любые$() йоты, заставляя их пропихнуться на стек при испо ...
+          "3": "Эта техника пригодится, если я захочу сослаться на специфичную йоту в своём $(#b38ef3)Заговоре/$, вроде сложного вектора или ссылки на сущность, без нужды создавать их с нуля при каждом исполнении.$(br2)Нужно не забыть о том, как именно я внесу такую йоту в список рун для исполнения. Простейшим способом будет сначала начертить "заглушку" при создании списка, а потом заменить её на ...
```
This should fix the build errors:
```suggestion
          "3": "Эта техника пригодится, если я захочу сослаться на специфичную йоту в своём $(#b38ef3)Заговоре/$, вроде сложного вектора или ссылки на сущность, без нужды создавать их с нуля при каждом исполнении.$(br2)Нужно не забыть о том, как именно я внесу такую йоту в список рун для исполнения. Простейшим способом будет сначала начертить \"заглушку\" при создании списка, а потом заменить её на нужную йоту с помощью руны $(l:patterns/lists#hexcasting:replace)$(action)Возвышения Хирурга/$.",
```
